### ISSUE TITLE
jsx-no-class-prop

### ISSUE THREAD

**Issue by benmosher** [yannickcr/eslint-plugin-react#70]
I routinely convert old jQuery-ish HTML snippets to React JSX and forget to `s/class=/className=/g`.

I was thinking this would be a relatively helpful React-specific view; report usage of `class` as a prop as it may be such an error. 

I'm up for writing it up and submitting a PR if there is interest/consensus.

for clarity, this is valid:

```jsx
var x = <div className="example"/>;
```
..and this would be reported:
```jsx
var y = <div class="example"/>;
```

..with some message like `Using class as a prop... did you mean 'className'?`

**Comment by yannickcr** [comment]
The [no-unknown-property](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md) rule handle this. This also warn you for the other HTML attributes that are different in JSX (like `for` -> `htmlFor`, `colspan` -> `colSpan`, etc.).

**Comment by benmosher** [comment]
Ah, sure enough. I poked around a little looking to see if this was already handled, but I only looked at the jsx-* rules. 

I will start using that one! 😀
